cli/lang-generator.js

```diff
@@ -0,0 +1,91 @@
+const fs = require('fs');
+const path = require('path');
+const sergeDirectories = require('../core.serge.json');
+
+const outputRoot = path.join(__dirname, '../generated');
+const outputPath = path.join(outputRoot, 'lang');
+
+function alphaSort(a, b) {
+	a = a.toLowerCase();
+	b = b.toLowerCase();
+	if (a < b) {
+		return -1;
+	} else if (a > b) {
+		return 1;
+	}
+	return 0;
+}
+
+function cleanDir(dirPath) {
```

Comment:
Looks like this is the same as the one in [icon-generator.js](https://github.com/BrightspaceUI/core/blob/master/cli/icon-generator.js#L41).  Perhaps we could move it out and share it?

Reply:
Good idea, `icon-generator.js` seems to do quite a bit with cleaning out the generated directory. Maybe adding a new clean script is the way to go as well.